test/functional/services/lib/leadfoot_element_wrapper.js

```diff
@@ -0,0 +1,314 @@
+   *
+   * @return {Promise<string>}
+   */
+  async getVisibleText() {
+    return await this._leadfootElement.getVisibleText();
+  }
+
+  /**
+   * Gets the tag name of the element. For HTML documents, the value is always lowercase.
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#getTagName
+   *
+   * @return {Promise<string>}
+   */
+  async getTagName() {
+    return await this._leadfootElement.getTagName();
+  }
+
+  /**
+   * Gets the position of the element relative to the top-left corner of the document,
+   * taking into account scrolling and CSS transformations (if they are supported).
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#getPosition
+   *
+   * @return {Promise<{x: number, y: number}>}
+   */
+  async getPosition() {
+    return await this._leadfootElement.getPosition();
+  }
+
+  /**
+   * Gets all elements inside this element matching the given CSS selector.
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#findAllByCssSelector
+   *
+   * @param {string} selector
+   * @return {Promise<LeadfootElementWrapper[]>}
+   */
+  async findAllByCssSelector(selector) {
+    return this._wrapAll(await this._leadfootElement.findAllByCssSelector(selector));
+  }
+
+  /**
+   * Gets the first element inside this element matching the given CSS selector.
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#findByCssSelector
+   *
+   * @param {string} selector
+   * @return {Promise<LeadfootElementWrapper>}
+   */
+  async findByCssSelector(selector) {
+    return this._wrap(await this._leadfootElement.findByCssSelector(selector));
+  }
+
+  /**
+   * Returns whether or not a form element can be interacted with.
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#isEnabled
+   *
+   * @return {Promise<boolean>}
+   */
+  async isEnabled() {
+    return await this._leadfootElement.isEnabled();
+  }
+
+  /**
+   * Gets all elements inside this element matching the given HTML tag name.
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#findAllByTagName
+   *
+   * @param {string} tagName
+   * @return {Promise<LeadfootElementWrapper[]>}
+   */
+  async findAllByTagName(tagName) {
+    return this._wrapAll(await this._leadfootElement.findAllByTagName(tagName));
+  }
+
+  /**
+   * Gets a property of the element.
+   * https://theintern.io/leadfoot/module-leadfoot_Element.html#getProperty
+   *
+   * @param {string} name
+   * @return {Promise<any>}
+   */
+  async getProperty(name) {
+    return await this._leadfootElement.getProperty(name);
+  }
+
+  /**
+   * Moves the remote environment’s mouse cursor to this element. If the element is outside
+   * of the viewport, the remote driver will attempt to scroll it into view automatically.
+   * https://theintern.io/leadfoot/module-leadfoot_Session.html#moveMouseTo
+   *
+   * @return {Promise<void>}
+   */
+  async moveMouseTo(...args) {
```

Comment:
Can we list the argument here explicitly and document then in the doc block too?